tests/metrics/test_classification.py

```diff
@@ -155,3 +155,30 @@ def test_when_no_interactions(self) -> None:
             expected_metric_per_user,
         )
         assert np.isnan(self.metric.calc(RECO, EMPTY_INTERACTIONS, CATALOG))
+
+
+class TestHitRate:
+    def setup(self) -> None:
+        self.metric = HitRate(k=2)
+
+    def test_calc(self) -> None:
+
+        # tp = pd.Series([1, 1, 0, 0])
+        # tn = pd.Series([6, 8, 7, 7])
+        # fp = pd.Series([1, 1, 2, 2])
+        # fn = pd.Series([2, 0, 1, 1])
+
+        expected_metric_per_user = pd.Series(
+            [1, 1, 0, 0],
+            index=pd.Series([1, 3, 4, 5], name=Columns.User, dtype=int), dtype=float
```

Comment:
Please execute `make lint` and `make test` before pushing the code to be sure everything is fine.  
To format the code automatically, run `make format`